### PR TITLE
Update/UI dashboard

### PR DESCRIPTION
**Pull request by youxuanpeng**
- made changes to room card component, please merge to avoid future conflict issues
- worked on the stepper component for the modal
image
- room card will still change in terms of FE code!

### CODE REVIEW

File: src/page/Home.jsx [hunk 1]
```diff
@@ -9,14 +9,16 @@ import {
 } from '../models/roomModel';
 import { getDbUser } from '../models/userModel';
 import RoomCard from '../components/RoomCard/RoomCard';
-import NewRoomModal from '../components/dashboard/NewRoomModal';
-import EditRoomModal from '../components/dashboard/EditRoomModal';
+import NewRoomModal from '../components/GeneralRoomModel/NewRoomModal';
+import EditRoomModal from '../components/GeneralRoomModel/EditRoomModal';
```
Comment (halfwhole) [inline src/page/Home.jsx:13]: just to check, are we going with `Model` for the folder name?

**Comment by halfwhole** [comment]
Looks good, though I'm seeing this for the 3rd step :(

Screenshot 2022-04-24 at 9 57 55 PM

Success is also kinda squished:

Screenshot 2022-04-24 at 9 59 37 PM

File: src/page/Home.jsx (hunk 1 above)
Comment (youxuanpeng) [reply src/page/Home.jsx:13]: oh shucks need to change it to modal! sorry!

**Comment by youxuanpeng** [comment]
maybe can wait till the end of today for me to make finish the changes for the general modal!

**Comment by youxuanpeng** [comment]
ok im done for this, you can merge it!

**Comment by halfwhole** [comment]
looks great! but will need to rebase this first to avoid undoing some newer changes

**Comment by halfwhole** [comment]
hang on sorry - I think just a few tweaks to this and should be ready to go!

**Comment by halfwhole** [comment]
Part of #27

**Pull request merged**